### DISCUSSION TITLE
Get linked collections and products from menu query

### DISCUSSION THREAD

**Discussion by bitforcesrl** [Shopify/storefront-api-feedback#161] [Ideas & Requests]
It's very common to add as menu item a collection or product, and we'd like to retrieve title desc and image for that collection or product. At the moment we've to do another query to get collection or products data after we retrieve the menu.

**Comment by gauravagarwal2704** [comment]
I observed that numerous developers have requested this feature. This [discussion thread](https://github.com/Shopify/storefront-api-feedback/discussions/116) for example addresses the same issue.

@morganmccunn @cocoahero @mutewinter

**Reply by bitforcesrl** [reply]
right i'll add on that thread this link as ref, thanks
